common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -62,6 +99,26 @@ static mon_wlan_hal::Event dwpal_to_bwl_event(const std::string &opcode)
     return mon_wlan_hal::Event::Invalid;
 }
 
+static mon_wlan_hal::Event dwpal_nl_to_bwl_event(uint8_t cmd, bool waiting_for_results_ready)
+{
+    if (cmd == NL80211_CMD_TRIGGER_SCAN) {
```

Comment:
I think that `switch-case` is more appropriate here.

Reply:
Added a switch-case